examples/grommet/src/components/PollResume/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { useVerna, cleanUiSchema } from '@openfun/verna';
+import { Button } from 'grommet';
+import './SectionResume.scss';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { schemaFormType } from '../../data/schemas';
+import Section from '../VernaOverride/Section';
+import QuestionResume from './QuestionResume';
+
+interface PollResumeProperties {
+  index: number;
+  onDelete: () => void;
+  poll: schemaFormType;
+  statsData: unknown;
+  setPollSelected: (poll: number | null) => void;
+}
+
+const messages = defineMessages({
+  delete: {
+    defaultMessage: 'Delete',
+    description: 'Label of the button to delete the section',
+    id: 'components.SectionResume.delete',
+  },
+  edit: {
+    defaultMessage: 'Edit',
+    description: 'Label of the button to edit the section',
+    id: 'components.SectionResume.edit',
+  },
+  view: {
+    defaultMessage: 'Answer',
+    description: 'Label of the button to view the section',
+    id: 'components.SectionResume.view',
+  },
+});
+
+export default function PollResume({
+  index,
+  onDelete,
+  poll,
+  setPollSelected,
+  statsData,
+}: PollResumeProperties) {
+  const { isEditor, setSchema, setUiSchema, setSchemaTranslations } = useVerna();
+  const { formatMessage } = useIntl();
+  const pollRJSF = poll.schema?.properties?.[Object.keys(poll.schema.properties)[0]];
+  const pollKey = Object.keys(poll.schema.properties || {})?.[0];
+
+  return (
+    <div className="form-section-resume">
+      <legend>{pollRJSF?.title && <FormattedMessage id={pollRJSF.title} />}</legend>
+      <p>
+        {pollRJSF?.description && <FormattedMessage id={pollRJSF.description} />}
+        <br />
+      </p>
+      {(poll.uiSchema?.[pollKey]?.['ui:order'] || []).map((question: string) => (
+        <QuestionResume
+          key={question}
+          poll={poll}
+          questionData={statsData?.[pollKey]?.[question] || {}}
+        />
+      ))}
+      <Button
+        primary
+        label={isEditor ? formatMessage(messages.edit) : formatMessage(messages.view)}
+        size="small"
+        style={{ marginTop: '10px' }}
+        onClick={() => {
+          setPollSelected(index);
+          setSchema(poll.schema);
+          setUiSchema(cleanUiSchema(poll.schema, poll.uiSchema, Section));
+          setSchemaTranslations(poll.translations);
+        }}
+      />
+      {isEditor && (
+        <Button
+          primary
+          label={formatMessage(messages.delete)}
+          onClick={onDelete}
+          size="small"
+          style={{ marginLeft: '10px', marginTop: '10px' }}
```

Comment:
```suggestion
          style={{ marginLeft: '10px', marginTop: '10px' }}
```
ditto there is a margin attribute for the Button element as well